docs/core/testing/unit-testing-mstest-sdk.md

```diff
@@ -229,3 +235,18 @@ Removing default package references:
 ```
 
 Finally, based on the extensions profile you're using, you can also remove some of the `Microsoft.Testing.Extensions.*` packages.
+
+### Updating your CI
+
+Once you have updated your projects, if you are using `MSTest runner` (default) and if you rely on `dotnet test` to run your tests, you will have to update your CI configuration. Please refer to [dotnet test integration](./unit-testing-platform-integration-dotnet-test.md#dotnet-test---microsofttestingplatform-mode) guide to understand all the changes needed.
+
+Here is an example of update when using `DotNetCoreCLI` task in Azure DevOps:
+
+```diff
+\- task: DotNetCoreCLI@2
+  inputs:
+    command: 'test'
+    projects: '**/**.sln'
+-    arguments: '--configuration Release'
++    arguments: '--configuration Release -p:TestingPlatformCommandLineArguments="--report-trx --coverage"'
```

Comment:
I found that the arguments line actually needs to be:

```
arguments: '--configuration Release -p:TestingPlatformCommandLineArguments="--report-trx --results-directory $(Agent.TempDirectory) --coverage"'
```

to fully replicate the previous functionality. (Hopefully that is a temporary change until the Azure task can be fixed!)

Reply:
```suggestion
+    arguments: '--configuration Release -p:TestingPlatformCommandLineArguments="--report-trx --results-directory $(Agent.TempDirectory) --coverage"'
```